File: app/src/main/java/com/fxn/pixsample/FragmentSampleActivity.java
package com.fxn.pixsample;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class FragmentSampleActivity extends AppCompatActivity {

@Override
protected void onCreate(Bundle savedInstanceState) {
	super.onCreate(savedInstanceState);
	setContentView(R.layout.activity_fragment_sample);
	getSupportFragmentManager().beginTransaction()
			.replace(R.id.container, SampleFragment.newInstance("", ""))
			.commit();
}
}

File: app/src/main/java/com/fxn/pixsample/FragmentSampleActivity.kt
package com.fxn.pixsample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class FragmentSampleActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_fragment_sample)

        supportFragmentManager.beginTransaction()
                .replace(R.id.container, SampleFragment.newInstance("", ""))
                .commit()
    }
}